packages/manager/src/managers/screenshots/ScreenshotsManager.ts

```diff
@@ -296,7 +296,7 @@ export class ScreenshotsManager extends BaseManager {
 		args: ScreenshotsManagerDeleteScreenshotFolderArgs,
 	): Promise<void> {
 		const res = await this._fetch({
-			body: { sliceId: args.sliceID },
+			body: { sliceName: args.sliceID },
```

Comment:
Maybe add a small comment that this is because it's the argument the ACL expects but that we do want a Slice ID here (because those aren't sensible to renaming)?